### PR TITLE
Greg/env

### PR DESCRIPTION
**Pull request by GodGreg**
Added code for env prod vs dev setup

npm run dev (runs the backend and frontend at the same time)

### CODE REVIEW

**Comment by nkxy** [comment]
this is a thicc pr

**Comment by GodGreg** [comment]
> nd at the same time)

File: backend/mongo.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      console.log("Mongo DB connection successful.");
+    }
+});
+
+
+// need this for promises
+mongoose.Promise = Promise;
+
+// require the schema
+//When you made a database call you will do something like
```
Comment (GodGreg) [inline backend/mongo.js]: make*

File: scripts/SVP.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
...evator Pitch and Vision - 1800 characters (~300 words),Senior Leadership - 900 characters (~150 words),Board of Directors - 600 characters (~100 words),Senior Leadership and Board Dynamics - 600 characters (~100 words),Expected Leadership Changes - 900 characters (~150 words),Succession Plan - 900 characters (~150 words),Capacity Building Areas and Recent Experience - 900 characters (~150 words),Potential Projects for SVP Partnership - 1500 characters (~250 words),Results and Impact - 900 characters (~150 words),Innovation and Approach  - 900 characters (~150 words),Organizational Challenges and Opportunities  - 900 characters (~150 words),Additional Information - 600 characters (~100 words),"A list of your current Board of Directors AND 3 external references (references could include other funders, past board members, or other community leaders etc)",An Executive Director bio,The two most recent audited financial statements,The current year's operational budget,"Updates to the financial statements provided above, including non-audited or audited YTD statements",A letter supporting this application signed by the Board Chair and Executive Director,Mission and Vision (100 characters)
```
Comment (GodGreg) [inline scripts/SVP.csv]: This is data downloaded from google forms to quickly seed the database

File: scripts/importApplications.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Seeds mongo database
+# Expects file SVP.csv in the same directory. Ask Greg or Karissa for this file.
+# Argument
+#   ${1} - Mongo db username
+#   ${2} - Mongo db password
+#   ${3} - Production or Development Database
```
Comment (GodGreg) [inline scripts/importApplications.sh]: It only takes 2 args right now - so we don't need line 8

File: src/App.js [hunk 4]
```diff
@@ -1,71 +1,166 @@
...
+import Application from "./Components/Application/Application";
+import ApplicationsTable from './Components/List/ApplicationList/ApplicationsTable';
+import Comparison from "./Components/Comparison/Comparison";
 
+import { ConnectedRouter } from "connected-react-router";
+import { ThemeProvider } from "@material-ui/core/styles";
 import theme from "./theme";
 import { history } from "./Store";
+import "./App.css"
+//import './App.css';
```
Comment (GodGreg) [inline src/App.js]: ok

File: src/App.js [hunk 5]
```diff
@@ -1,71 +1,166 @@
...
 
+//Use this later for prod vs dev environment
+//// TODO: Uncomment when express is setup
+const proxy = process.env.NODE_ENV === "production" ? process.env.REACT_APP_SERVER : "http://localhost:4000";
```
Comment (GodGreg) [inline src/App.js]: prod is not currently hosted anywhere (Todo)

**Pull request merged**